tensorflow-sys/build.rs

```diff
@@ -94,13 +97,43 @@ fn remove_suffix(value: &mut String, suffix: &str) {
     }
 }
 
+#[cfg(not(target_env = "msvc"))]
 fn extract<P: AsRef<Path>, P2: AsRef<Path>>(archive_path: P, extract_to: P2) {
     let file = File::open(archive_path).unwrap();
     let unzipped = GzDecoder::new(file);
     let mut a = Archive::new(unzipped);
     a.unpack(extract_to).unwrap();
 }
 
+#[cfg(target_env = "msvc")]
+fn extract<P: AsRef<Path>, P2: AsRef<Path>>(archive_path: P, extract_to: P2) {
+    fs::create_dir_all(&extract_to)
+        .expect("Failed to create output path for zip archive.");
+    let file = File::open(archive_path)
+        .expect("Unable to open libtensorflow zip archive.");
+    let mut archive = ZipArchive::new(file).unwrap();
+    for i in 0..archive.len() {
+        let mut zipfile = archive.by_index(i).unwrap();
+        let output_path = extract_to.as_ref().join(zipfile.sanitized_name());
+        if (&*zipfile.name()).starts_with("lib") {
```

Comment:
Can't this be simplified to just `zipfile.name().starts_with("lib")`?

Reply:
It sure can. Can't recall what sequence of silliness led me to that anyway. :D